content/doc/administration/requirements/java.adoc

```diff
@@ -15,7 +15,7 @@ Modern Jenkins versions have the following Java requirements:
 ** See the link:/doc/administration/requirements/upgrade-java-guidelines[Java 8 to Java 11 upgrade guidelines] for additional upgrade instructions
 * Java 7 and prior are not supported
 * Java 9 and 10 are not supported
-* Java 12, 13, 14, 15, and 16 are not supported
+* Java 12, 13, 14, 15, 16, and 17 are not supported
```

Comment:
not strictly true, Java 17 is available for preview in the docker images, Mark planned to write a blog post but hasn't had time.

https://hub.docker.com/layers/jenkins/jenkins/jdk17-preview/images/sha256-3be2044ec2d27b3d807e2597da99d5eb7b9beda33ca549e456f5150e72166e67?context=explore

and the agent images too

Reply:
maybe? probably deserves a bigger entry / blog post link

```suggestion
* Java 12, 13, 14, 15, and 16 are not supported
* Java 17 is available for early adopters when using the Docker images (`jenkins/jenkins:jdk17-preview`)
```